linode/volumes/framework_models.go

```diff
@@ -0,0 +1,41 @@
+package volumes
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/linode/linodego"
+	"github.com/linode/terraform-provider-linode/linode/helper/frameworkfilter"
+	"github.com/linode/terraform-provider-linode/linode/volume"
+)
+
+type VolumeModel struct {
```

Comment:
nit: probably better to rename it to be `VolumeFilterModel`, which is aligned with the naming in our code base.